src/config.js

```diff
@@ -145,3 +145,36 @@ export const errorMessages = {
 export const datasetStatLabels = ['rows', 'columns', 'file_size'];
 
 export const statsRowLen = 33;
+
+export const s3BucketRegions = [
+  'us-east-2',
+  'us-east-1',
+  'us-west-1',
+  'us-west-2',
+  'af-south-1',
+  'ap-east-1',
+  'ap-south-2',
+  'ap-southeast-3',
+  'ap-southeast-4',
+  'ap-south-1',
+  'ap-northeast-3',
+  'ap-northeast-2',
+  'ap-southeast-1',
+  'ap-southeast-2',
+  'ap-northeast-1',
+  'ca-central-1',
+  'cn-north-1',
+  'cn-northwest-1',
+  'eu-central-1',
+  'eu-west-1',
+  'eu-west-2',
+  'eu-south-1',
+  'eu-west-3',
+  'eu-north-1',
+  'eu-south-2',
+  'eu-central-2',
+  'sa-east-1',
+  'me-south-1',
+  'me-central-1',
+  'il-central-1',
+];
```

Comment:
I think this is fine for now. But, it would be nice if we can use some sort of library like boto3 in the backend and return the list of available regions instead of hardcoding in the FE. Since AWS regions might be changed over time.